src/NIST-800-53/nist-800-53.ts

```diff
@@ -521,6 +542,74 @@ export class NIST80053Checks extends NagPack {
         this.createMessage(ruleId, info, explanation)
       );
     }
+    if (
+      !this.ignoreRule(ignores, 'NIST.800.53-RDSEnhancedMonitoringEnabled') &&
+      !nist80053RDSEnhancedMonitoringEnabled(node)
+    ) {
+      const ruleId = 'NIST.800.53-RDSEnhancedMonitoringEnabled';
+      const info =
+        'The RDS DB Instance does not enhanced monitoring enabled- (Control ID: CA-7(a)(b)).';
+      const explanation =
+        'Enable Amazon Relational Database Service (Amazon RDS) to help monitor Amazon RDS availability. This provides detailed visibility into the health of your Amazon RDS database instances.';
+      Annotations.of(node).addError(
+        this.createMessage(ruleId, info, explanation)
+      );
+    }
+    if (
+      !this.ignoreRule(
+        ignores,
+        'NIST.800.53-RDSInstanceDeletionProtectionEnabled'
+      ) &&
+      !nist80053RDSInstanceDeletionProtectionEnabled(node)
+    ) {
+      const ruleId = 'NIST.800.53-RDSInstanceDeletionProtectionEnabled';
+      const info =
+        'The RDS DB Instance does not have deletion protection enabled - (Control ID: SC-5).';
+      const explanation =
+        'Ensure Amazon Relational Database Service (Amazon RDS) instances have deletion protection enabled. Use deletion protection to prevent your Amazon RDS instances from being accidentally or maliciously deleted, which can lead to loss of availability for your applications.';
```

Comment:
```suggestion
        'Ensure Amazon Relational Database Service (Amazon RDS) instances  and clusters have deletion protection enabled. Use deletion protection to prevent your Amazon RDS instances and clusters from being accidentally or maliciously deleted, which can lead to loss of availability for your applications.';
```